Packs/SplunkPy/ReleaseNotes/2_2_2.md

```diff
@@ -0,0 +1,8 @@
+
+#### Scripts
+##### SplunkShowIdentity
+- Update returned value in case of an error
+##### SplunkShowAsset
+- Update returned value in case of an error
+##### SplunkShowDrilldown
+- Update returned value in case of an error
```

Comment:
```suggestion
- Improved the readable output returned in case of an error.
##### SplunkShowAsset
- Improved the readable output returned in case of an error.
##### SplunkShowDrilldown
- Improved the readable output returned in case of an error.
```